tests/unit/amo/components/TestSearchPage.js

```diff
@@ -48,44 +48,49 @@ describe('<SearchPage />', () => {
 
   it('renders a Paginate', () => {
     const root = render();
-    const paginator = findByTag(root, Paginate);
-    expect(paginator.props.count).toEqual(80);
-    expect(paginator.props.currentPage).toEqual(3);
-    expect(paginator.props.pathname).toEqual('/search/');
-    expect(paginator.props.queryParams).toEqual({ q: 'foo' });
+    const paginator = root.find(Paginate);
+    expect(paginator.prop('count')).toEqual(80);
```

Comment:
I agree re: them sounding similar. It's quite hard to stop just using `toEqual()` everywhere.